src/puppetlabs/trapperkeeper/services.clj

```diff
@@ -22,6 +22,8 @@
   (service-context [this] "Returns the context map for this service")
   (get-service [this service-id] "Returns the service with the given service id")
   (get-services [this] "Returns a sequence containing all of the services in the app")
+  (call-service-fn [this other-service-id fn-name args default] "Calls named function in other service, returning default if that service is not included.")
+  (service-included? [this service-id] "Returns nil if service does not exist and the service if it does")
```

Comment:
I would expect that this is exactly what `get-service` already does, but since you added this, I'm assuming that `get-service` throws if the service doesn't exist?  Perhaps it should simply be changed?